blackboxopt/evaluation.py

```diff
@@ -106,6 +106,10 @@ def __len__(self):
     def to_json(self, **json_dump_kwargs):
         return json.dumps(asdict(self), **json_dump_kwargs)
 
+    # TODO: This could be an attribute, a method called dict or to_dict, what to go for?
+    def dict(self):
```

Comment:
I think my favorite would be `to_dict()`, like in `pandas.DataFrame`.

Reply:
I like the additional pandas convention argument, thanks :relaxed: